src/Core/Project.cs

```diff
@@ -73,7 +73,7 @@ public Project()
         /// </param>
         public void FireScriptEvent(ScriptEvent @event)
         {
-            foreach (var scriptFile in ScriptFiles)
+            foreach (var scriptFile in ScriptFiles.ToList())
```

Comment:
`ConcurrentObservableCollection.GetEnumerator` already calls `ToList()`, is there a reason to do it a second time?

Reply:
Done in 140746e